build.sbt

```diff
@@ -42,7 +41,7 @@ lazy val microservice = Project(appName, file("."))
     addCompilerPlugin("org.typelevel"  %% "kind-projector"  % "0.13.0" cross CrossVersion.full),
     addCompilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full)
```

Comment:
We can use this version
`"silencer-lib" % "1.7.5"`

Reply:
fixed versions